string name
float32 version
JointDescriptionM[] joints
